package.json

```diff
@@ -24,12 +23,13 @@
     "lodash": "4.17.21",
     "prop-types": "^15.7.2",
     "qs": "6.9.3",
+    "react-native-device-info": "~8.4.8",
```

Comment:
Let's zapeči here. This package should be safe in combination with the RN version, but let's be sure

Reply:
Were it any other dep, I would agree, but this one regularly receives new devices to and info about them (specifically we're interested in `hasNotch` list).

I've checked, they've not had an instance of patch versions causing any issues, they take semver seriously.

Unless you insist, I'd really like to keep the `~`.